sunkit_pyvista/plotter.py

```diff
@@ -29,7 +30,8 @@ def __init__(self, coordinate_frame=None):
         if coordinate_frame is None:
             coordinate_frame = HeliocentricInertial()
         self._coordinate_frame = coordinate_frame
-        self._plotter = pv.Plotter(off_screen=True)
+        self._plotter = pv.Plotter()
+        self._plotter.show_axes()
```

Comment:
What's the motivation behind this? I think it might be better to not show the axes, but open to reasons why we should keep them.

Reply:
I was using it for displaying the plots and trying to figure out how the rotation works, we can get rid of them